cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -157,6 +157,24 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         // AssessmentQuestionnaire.delete(cloudNative);
     });
 
+    it("Discards application assessment through Assessment Actions page", function () {
+        // Polarion TC MTA-440
+        AssessmentQuestionnaire.enable(legacyPathfinder);
```

Comment:
This is already enabled in before method .

Reply:
this was disabled in a TC before it (it("View archived questionnaire", function () {) and was not enabled again
i will enable it in the above TC and remove this line of code from current TC

Reply:
If it is needed by all TC's then we can put it in "beforeEach" method

Reply:
some TC's purposely disables it as it is part of the test case // this PR has been closed and opened new one that merged two polarion TC'S